### ISSUE TITLE
Aditional languages (rus, ukr) doesn't choosing in OCR language list

### ISSUE THREAD

**Issue by deimjons** [ciur/papermerge#612]
**Description**
Hello. 
I am using a custom docker image with Russian and Ukrainian language packages for tesseract installed, following instructions: https://docs.papermerge.io/3.0/setup/add-ocr-langs/
Dockerfile:

```
FROM papermerge/papermerge:3.1

# add Ukrainian and Russian OCR languages
RUN apt install tesseract-ocr-rus tesseract-ocr-ukr
```

**Info:**
 - Papermerge Version [e.g. 3.1]
 In the container are added languages:
```
# docker exec -it papermerge bash
# tesseract --list-langs
List of available languages in "/usr/share/tesseract-ocr/5/tessdata/" (11):
deu
eng
fra
ita
nld
osd
por
ron
rus
spa
ukr
```
I see these languages in OCR languages list but can not choose and **Run OCR**

<img width="1710" alt="Screenshot 2024-04-19 at 18 45 39" src="https://github.com/ciur/papermerge/assets/10740038/b055abff-f5fa-4332-93c3-dbbad86c636b">

**Comment by ciur** [comment]
Yes, because "rus" and "ukr" language codes are missing in following places:

-   https://github.com/papermerge/papermerge-core/blob/master/ui/src/types.ts#L269
-   https://github.com/papermerge/papermerge-core/blob/master/ui/src/cconstants.ts#L10

I would gladly accept your pull request.

**Comment by bl1nkker** [comment]
@deimjons This issue is related to https://github.com/ciur/papermerge/issues/624, where I have provided detailed steps on how to add new OCR languages to Papermerge

**Comment by ciur** [comment]
Docs were update with steps how to add additional language: https://docs.papermerge.io/latest/setup/add-ocr-langs/
